cleanlab/count.py

```diff
@@ -1458,7 +1455,7 @@ def get_confident_thresholds(
         #  this approach is that there will be no standard value returned for missing classes.
         labels = labels_to_array(labels)
         all_classes = range(pred_probs.shape[1])
-        unique_classes = get_unique_classes(labels)
+        unique_classes = get_unique_classes(labels, multi_label=multi_label)
```

Comment:
This is the only code change in this file; everything else is just sorting the import statements.

I think this is a smart move 👍